docs/contrib/modifications.md

```diff
@@ -14,6 +19,8 @@ Mode](https://firefox-source-docs.mozilla.org/devtools-user/responsive_design_mo
 or [Chrome/Chromium Device
 Mode](https://developer.chrome.com/docs/devtools/device-mode/) to
 configure your browser with 1080p.
+Screen shots should be added to a directory named `assets` that is a
```

Comment:
I am not sure, but I would bet "screenshots" (one word) is better -- at least for our readership.